docs/features/pieces-copilot.mdx

```diff
@@ -116,3 +116,48 @@ Our Copilot also comes with multiple different LLM runtimes, cloud and local in
 <Info>*Local LLMs come with some hefty hardware requirements, please make sure your system can fulfill them before attempting to load one of these models.*</Info>
 - Llama2, a model trained by Meta AI optimized for completing general tasks, also available in a CPU and GPU runtime.
     - *These models require 5.6GB of RAM and 5.6GB of VRAM for the cpu and gpu runtimes respectively.*
+
+## Plugin Integrations
+
+Our VS Code, Jetbrains, Obsidian, and Jupyter Lab integrations all offer the same copilot chat features, ensuring a consistently powerful experience wherever you need it.
```

Comment:
Also include PWE

Reply:
@mason-at-pieces

Reply:
The one thing here is that I don't know if Web Extension has all the features of copilot in the the plugins. I've added it though